x-pack/plugins/ml/server/plugin.ts

```diff
@@ -87,7 +87,10 @@ export class MlServerPlugin implements Plugin<MlPluginSetup, MlPluginStart, Plug
       navLinkId: PLUGIN_ID,
       app: [PLUGIN_ID, 'kibana'],
       catalogue: [PLUGIN_ID],
```

Comment:
We should declare ML's management app as part of the feature as well. Not _strictly_ required as part of this PR, but it'll help us implement https://github.com/elastic/kibana/issues/35965

```suggestion
      management: {
        insightsAndAlerting: ['jobsListLink`]
      },
      catalogue: [PLUGIN_ID],
```